polygeist/tools/cgeist/Test/polybench/datamining/correlation/correlation.c

```diff
@@ -1,19 +1,19 @@
 // RUN: clang %s -O3 %stdinclude %polyverify -o %s.exec1 -lm && %s.exec1 &> %s.out1
-// RUN: cgeist %s %polyverify %stdinclude -O3 -o %s.execm && %s.execm &> %s.out2
+// RUN: cgeist -omit-fp-contract %s %polyverify %stdinclude -O3 -o %s.execm && %s.execm &> %s.out2
```

Comment:
Could we make our life easier by only enabling FP contraction for C++, but not for C?

Reply:
I'd say we go with the clang defaults, we introduced this to not differ with them in the first place. I just didn't wanna update all the tests here as we're already checking in `fmuladd.cpp`. WDYT?

Reply:
Yeah, makes sense 👍